internal/services/recoveryservices/recovery_services_vault_resource.go

```diff
@@ -99,6 +99,7 @@ func resourceRecoveryServicesVault() *pluginsdk.Resource {
 			"sku": {
 				Type:             pluginsdk.TypeString,
 				Required:         true,
+				ForceNew:         true,
```

Comment:
could you not just add a separate patch call in the update to handle the sku?

Reply:
`sku` can only be changed by `CreateOrUpdate`, not the `Update`, and `encryption` property is only available in the `Update`. When `encryption` is specified in the initial create, we are not able to call the `CreateOrUpdate` again to update `sku` because it tries removing the encryption which is not allowed by the service

Reply:
but it sounds like you can change the `sku` with a put during update if `encryption` has not been specified?